rakelib/test.rake

```diff
@@ -17,6 +17,18 @@ namespace "test" do
     require "rspec/core/runner"
     require "rspec"
     require 'ci/reporter/rake/rspec_loader'
+    setup_json_logging
+  end
+
+  def setup_json_logging
+    require "cabin"
+    require "logstash/logging/json"
+    Cabin::Channel.get(LogStash).tap do |logger|
+      # TODO(sissel): Open null on windows?
+      out = File.new("/dev/null", "a")
+      logger.subscribe(LogStash::Logging::JSON.new(out))
+      logger.level = :debug
+    end
```

Comment:
This will surely fails on windows, we can make it cross platform if we can create a dummy io object that implement the minimal interface to be an IO object but doesn't actually write anything to disk.

OR we can use this, which will make a null stream, I believe it will work on windows but I havent tested it?
```
File.open(File::NULL, "w")
```